src/xenia/memory.cc

```diff
@@ -716,6 +716,10 @@ void BaseHeap::DumpMap() {
   }
 }
 
+uint32_t BaseHeap::GetUsedPagesCount() {
```

Comment:
Inconsistent usage of singular/plural words. We use "PageCount", not "PagesCount", in the code base.

Reply:
Good catch 🐱 Thanks